bundles/automation/org.eclipse.smarthome.automation.sample.extension.json/SmartHome Runtime Sample Rules.launch

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Then the git history lies ;)

`git show --stat --format=raw e1701808da980cd24c436417c032c9679132464a`

```
commit e1701808da980cd24c436417c032c9679132464a
tree d942310d1d2441a28d6b3a86e5158c40b0e90cd6
parent 93b00d106b5839feb6143b532e7c003590957e36
author Ana Dimova <a.dimova@prosyst.bg> 1467962736 +0300
committer Ana Dimova <a.dimova@prosyst.bg> 1467962736 +0300

    Refactor #1569 implemetation
    
    Signed-off-by: Ana Dimova <a.dimova@prosyst.bg>

 bundles/automation/.project                                                                                                                                         |  17 +++++++++
 bundles/automation/.settings/org.eclipse.core.resources.prefs                                                                                                       |   2 ++
 bundles/automation/.settings/org.eclipse.m2e.core.prefs                                                                                                             |   4 +++
 bundles/automation/org.eclipse.smarthome.automation.importers/src/main/java/org/eclipse/smarthome/automation/internal/importers/AbstractImporter.java               | 124 ---------------------------------------------------------------
 bundles/automation/org.eclipse.smarthome.automation.importers/src/main/java/org/eclipse/smarthome/automation/internal/importers/Activator.java                      | 164 -----------------------------------------------------------------------------------
 bundles/automation/org.eclipse.smarthome.automation.importers/src/main/java/org/eclipse/smarthome/automation/internal/importers/ModuleTypeImporter.java             | 215 ------------------------------------------------------------------------------------------------------------
 bundles/automation/org.eclipse.smarthome.automation.importers/src/main/java/org/eclipse/smarthome/automation/internal/importers/TemplateImporter.java               | 208 ---------------------------------------------------------------------------------------------------------
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/.classpath                                        |   3 +-
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/.project                                          |   2 +-
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/META-INF/MANIFEST.MF                              |  10 +++---
 bundles/automation/org.eclipse.smarthome.automation.provider.file/OSGI-INF/ModuleTypeFileProvider.xml                                                               |  18 ++++++++++
 bundles/automation/org.eclipse.smarthome.automation.provider.file/OSGI-INF/TemplateFileProvider.xml                                                                 |  18 ++++++++++
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/about.html                                        |   0
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/build.properties                                  |   0
 bundles/automation/{org.eclipse.smarthome.automation.importers => org.eclipse.smarthome.automation.provider.file}/pom.xml                                           |   0
 bundles/automation/org.eclipse.smarthome.automation.provider.file/src/main/java/org/eclipse/smarthome/automation/internal/provider/file/AbstractFileProvider.java   | 289 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 bundles/automation/org.eclipse.smarthome.automation.provider.file/src/main/java/org/eclipse/smarthome/automation/internal/provider/file/ModuleTypeFileProvider.java |  92 +++++++++++++++++++++++++++++++++++++++++++++++
 bundles/automation/org.eclipse.smarthome.automation.provider.file/src/main/java/org/eclipse/smarthome/automation/internal/provider/file/TemplateFileProvider.java   |  91 ++++++++++++++++++++++++++++++++++++++++++++++
 bundles/automation/org.eclipse.smarthome.automation.sample.extension.json/SmartHome Runtime Sample Rules.launch                                                     |  34 ------------------
 19 files changed, 539 insertions(+), 752 deletions(-)
```